### PR TITLE
Full website

### PR DESCRIPTION
**Pull request by ONJoseph**
- [ ] Add maths icon and images 
- [ ] Implemented calculator functionality
- [ ] Implemented quote functionality
- [ ] Add calculator styling
- [ ] Add quote styling
- [ ] modified package.json file
- [ ] Updated README.md file

### CODE REVIEW

File: src/components/Calculator.css [hunk 1]
```diff
@@ -1,9 +1,11 @@
 .container {
-  margin-top: 4rem;
-  margin-left: 50%;
   display: flex;
-  flex-direction: column;
-  max-width: 33.33%;
+  justify-content: space-between;
+  padding: 0 2rem;
+}
+
```
Comment (Mihndim2020) [inline src/components/Calculator.css:6]: - [x] As it is a [requirement ](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_full_website.md)for this project, " `The general layout should match these wireframes:` ", please make sure your calculator page matches the wireframe. Please make sure all the text and layout of your calculator page is the same as in the [wireframe](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/images/math_magicians_calculator.png).

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi @ONJoseph, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file
:heavy_check_mark: Clean and organized code

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION

# Status: Requires Changes ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can get to me on slack  using this handle @mihndim.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.css (hunk 1 above)
Comment (ONJoseph) [reply src/components/Calculator.css:6]: Done. Styling meets project wireframe template. Thanks.

**Review by NduatiKagiri — APPROVED** [review]
Hi @ONJoseph 👋 ,

# APPROVED 💯  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

ea0cb5a48e3337523ffe8d60f177879d

## Highlights ✔️ 
✅ GitHub Flow observed
✅ No Linters errors
✅ Descriptive README
✅ Good project description

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**